.tekton/manual-approval-gate-next-webhook-push.yaml

```diff
@@ -4,7 +4,7 @@ kind: PipelineRun
 metadata:
   annotations:
     pipelinesascode.tekton.dev/cancel-in-progress: "true" # Cancel in-progress pipelines
-    pipelinesascode.tekton.dev/pipeline: "https://raw.githubusercontent.com/openshift-pipelines/operator/refs/heads/next/.tekton/docker-build-ta.yaml"
+    pipelinesascode.tekton.dev/pipeline: "https://raw.githubusercontent.com/openshift-pipelines/operator/refs/heads/release-v0.4.1/.tekton/docker-build-ta.yaml"
```

Comment:
@vdemeester @pramodbindal I think the branch for annotation need to be handled different way for manual-approval-gate

Reply:
can we just keep `docker-build-ta.yaml` in main branch and refer from there instead of maintaining branch vise